geotrek/signage/tests/test_command_signage.py

```diff
@@ -206,7 +207,7 @@ def test_update_same_eid(self):
         call_command('loadsignage', filename, eid_field='eid', type_default='label',
                      name_default='name', verbosity=2, stdout=output)
         self.assertIn("Update : name with eid eid1", output.getvalue())
-        self.assertEqual(Signage.objects.count(), 2)
+        self.assertEqual(Signage.objects.count(), 3)
```

Comment:
Can you add an assertion here to check that the signage has te right conditions associated please ?

Reply:
Still need the assertion please